scalajslib/test/src/mill/scalajslib/FullOptESModuleTests.scala

```diff
@@ -29,7 +29,7 @@ object FullOptESModuleTests extends TestSuite {
 
     test("fullOpt with ESModule moduleKind") {
       val result =
-        fullOptESModuleModuleEvaluator(FullOptESModuleModule.fullOptESModuleModule.fullOpt)
+        fullOptESModuleModuleEvaluator(FullOptESModuleModule.fullOptESModuleModule.fullLinkJS)
```

Comment:
Hard to tell, whether this was intentional. But if we are going to test `fullLinkJS` instead of the deprecated `fullOpt`, we should also rename the test case and the involved `val`s and `object`s. They currently all contain the `fullOpt`.